quay-enterprise/configure-machines.md

```diff
@@ -1,15 +1,74 @@
-# Configure Machines for Enterprise Registry
+# Configure Machines for Quay Enterprise
 
-The Enterprise Registry allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
+Quay Enterprise allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
 
 This guide we will assume you have the DNS record `registry.example.com` configured to point to your Enterprise Registry.
```

Comment:
This guide assumes the DNS A record `registry.example.com` points to your Quay Enterprise Registry.

("Quay" may have been intentionally committed in your version and if so, OK, omit it here.